docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -324,6 +324,35 @@ and needs to be manually advanced):
   links to the security registrar (http://www.silverstripe.org/download/security-releases)
   match the pages saved in draft.
 
+#### Basing a new release on a previous one (tweak releases)
+
+Commonly a stable release will need to mirror the contents of the release
+candidate that preceded it, sometimes with a small set of additional commits.
+However, running the standard `cow release` command will create a release that
+includes all the latest commits on the branches it targets, which can include
+unaudited code. A **tweak release** includes only the commits present in the
+previous tagged release by default, and can optionally include additional
+commits when necessary. To create one, use the `release:checkout-tagged-base`
+command:
+
+1. `cow release:create <new-version>` to create the new release.
+2. `cow release:plan <new-version>` to generate a plan for the new release.
+3. `cow release:checkout-tagged-base <new-version>` to shift all of the modules
```

Comment:
how is the `previous version tag` defined?

Reply:
When we run Cow commands, we always specify `<new-version>` so it know which folder to run in / what version to work against. `previous version tag` here literally refers to the last version tag on the module we're releasing from (`silverstripe/installer` for core), which is what the `checkout-tagged-base` command uses implicitly.

Reply:
ok in that case case we change the x2 `previous version tag` to `most recently applied tag` to make it a little clearer?

Reply:
I don't think 'most recent' describes it correctly either - if we were to produce a new tag on an older release branch (e.g. 4.4.5, when we're trying to release 4.5.1), this command wouldn't use that tag. I've adjusted the wording around this step to try and clarify - let me know what you think.

Reply:
I'd propose changing a few words around - is this correct?
**How?** This command finds the last common commit between the latest tag on
    the chosen branch and the tip of that branch, and then shifts the HEAD to
    that commit.

Reply:
Tweaked 👍